### ISSUE TITLE
Support building without embedded libbacktrace

### ISSUE THREAD

**Issue by jhance** [vmprof/vmprof-python#248]
Our python setup requires cross compiling, which isn't working because of `./configure` running within `setup.py`. I have locally patched out the embedded `libbacktrace` and replaced it with link opts to a cross compiled `libbacktrace`. It would be nice to not need the patch; perhaps you would accept an upstreamable patch to have an environment var that drops the `libbacktrace` embed stuff (and maybe append `-lbacktrace`)?

**Comment by mattip** [comment]
Do you mean you are using the `libbacktrace` sources in this repo to build a cross compiled version without `configure`? Could we adopt the direct call to the compiler/linker for all the builds without `configure`?

**Comment by jhance** [comment]
No, I actually built it from upstream source and linked it statically in with some flag (iirc `--extra-link-objects` or something).

**Comment by mattip** [comment]
The change would be to have a configuration option to use the native `libbacktrace` ?
